code/modules/supermatter/supermatter.dm

```diff
@@ -162,6 +163,9 @@
 			empulse(get_turf(src), 100, 200, 1)
 			qdel(src)
 			return
+		for(var/mob/M in player_list)
+			M.playsound_local(src, 'sound/effects/delamination.ogg', 50, 1)
```

Comment:
I'm not sure this is the implementation you want -- it might be but it's unclear. Do you want them to only hear this sound if they are normally able to hear? That is to say, not in space, not deaf, etc.?

1. If you want all players to hear it no matter what, even if deaf or in space, use M << 'sound/path.ogg' instead.
2. If you want only players who are reasonably able to hear it, you should probably at least filter players who are not on z1

Reply:
My original intent was just the sound to be heard on Z1, but it seems people really liked the idea of it being heard on all Z levels. It is a fragment of reality bending instability after all. Should deaf people be included you think?

Reply:
Your call

Reply:
> My original intent was just the sound to be heard on Z1, but it seems people really liked the idea of it being heard on all Z levels, so I left it like that. It is a fragment of reality bending instability after all. Should deaf people be included you think?

yes
project zomboid deafness playthrough, where you enjoy the sounds of your rock and roll music until a 112 decibel vine boom sound effect erupts because you saw a zombie standing in a window across the street